app/src/main/java/com/example/loic/comics_app_android/presentation/ui/adapter/RecyclerComicsListAdapter.java

```diff
@@ -1,4 +1,59 @@
 package com.example.loic.comics_app_android.presentation.ui.adapter;
 
-public class RecyclerComicsListAdapter {
+import android.support.v7.widget.RecyclerView;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.example.loic.comics_app_android.R;
+import com.example.loic.comics_app_android.data.model.ResultsItem;
+import com.example.loic.comics_app_android.presentation.ui.viewholder.ComicsListHolder;
+import com.squareup.picasso.Picasso;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.List;
+
+public class RecyclerComicsListAdapter extends RecyclerView.Adapter<ComicsListHolder> {
+    private List<ResultsItem> itemList;
+
+    //Constructeur à adapter selon les données
+    public RecyclerComicsListAdapter(List<ResultsItem> argDataset) {
```

Comment:
Renommer le nom de la variable en paramètre en `listComics` pour comprendre ce quelle reçois

Reply:
Plus simple, j'ai changé itemList en "comicList" et pareil pour argDataset (puis this.comicList = comicList dans le constructeur)
✔️